### DISCUSSION TITLE
Default binding for "/" (slash) key to search

### DISCUSSION THREAD

**Discussion by gonzus** [LazyVim/LazyVim#1410] [General]
When I press "/" to search for something, a small Q-like icon shows on the bottom-left corner of my screen. Things usually work the same as in stock vim, but sometimes (maybe when I type too fast?) the search exits and my keys are received by the buffer, meaning that I can easily change the text without meaning to.

Is this a separate plugin? I could not find any documentation for "/" on https://www.lazyvim.org/keymaps.

Anybody else has seen this same behaviour, exiting the search box before pressing enter? Is anything suggested to avoid this behaviour?

**Comment by dpetka2001** [comment]
This is from [flash.nvim](https://github.com/folke/flash.nvim) plugin. You have 2 options:
1. Disable this feature entirely
```lua
return {
	{
  		"folke/flash.nvim",
  		opts = {
    		modes = {
      			search = {
        			enabled = false,
      			},
    		},
  		},
	},
}
```
2. Press `<C-s>` while in search mode to temporarily disable this feature in case you want to use it another time.
